test/models/TraderProfileSpec.scala

```diff
@@ -529,4 +529,76 @@ class TraderProfileSpec extends AnyFreeSpec with Matchers with TryValues with Op
       }
     }
   }
+
+  ".validateNirmsNumber" - {
```

Comment:
both validate nirms and build nirms do the same thing - you only need one

Reply:
I will check if I can use buildNirms instead :)